wasm/test_page/js/index.js

```diff
@@ -38,22 +38,48 @@ const _prepareTranslateOptions = (paragraphs) => {
   return translateOptions;
 };
 
+const textToHTML = (text) => {
+  const div = document.createElement('div');
+  div.appendChild(document.createTextNode(text));
+  return div.innerHTML;
+};
+
 const translateCall = () => {
-  const text = document.querySelector("#input").value + "  ";
+  const text = document.querySelector("#input").value;
   if (!text.trim().length) return;
 
-  const paragraphs = text.split("\n");
+  const paragraphs = [textToHTML(text)]; // escape HTML 
   const translateOptions = _prepareTranslateOptions(paragraphs);
-  $("#output").setAttribute("disabled", true);
```

Comment:
Why was this removed?

Reply:
I changed #output to be a `div` instead of a `textarea`, and div's don't have a disabled state.